test/e2e/common/node/sysctl.go

```diff
@@ -199,6 +155,27 @@ var _ = SIGDescribe("Sysctls [LinuxOnly] [NodeFeature:Sysctls]", func() {
 		gomega.Expect(err.Error()).NotTo(gomega.ContainSubstring("kernel.shmmax"))
 	})
 
+	ginkgo.It("should forbid unsafe sysctls [MinimumKubeletVersion:1.21]", func() {
+		pod := testPod()
+		pod.Spec.SecurityContext = &v1.PodSecurityContext{
+			Sysctls: []v1.Sysctl{
+				{
+					Name:  "net.core.somaxconn",
+					Value: "1024",
+				},
+			},
+		}
+		pod.Spec.Containers[0].Command = []string{"/bin/sysctl", "net.core.somaxconn"}
+
+		ginkgo.By("Creating a pod with the net.core.somaxconn sysctl")
+		pod = podClient.Create(pod)
+
+		ginkgo.By("Wait for pod failed reason")
+		// watch for pod failed reason instead of termination of pod
+		err := waitForPodFailedReason(f, pod, "SysctlForbidden")
+		framework.ExpectNoError(err)
+	})
+
 	ginkgo.It("should not launch unsafe, but not explicitly enabled sysctls on the node [MinimumKubeletVersion:1.21]", func() {
```

Comment:
- case 1: should forbid unsafe sysctls
- case 2: should not launch unsafe, but not explicitly enabled sysctls on the node 

It may be a little dup.

Reply:
Don't we want to still keep an e2e that explicitly permitted, unsafe sysctls are working?